engines/docker/config.go

```diff
@@ -0,0 +1,22 @@
+package dockerengine
+
+import (
+	schematypes "github.com/taskcluster/go-schematypes"
+	"github.com/taskcluster/taskcluster-worker/runtime/util"
+)
+
+type configType struct {
+	DockerSocket string `json:"dockerSocket"`
+}
+
+var configSchema = schematypes.Object{
+	Properties: schematypes.Properties{
+		"dockerSocket": schematypes.String{
+			Title: "Docker Socket",
+			Description: util.Markdown(`Path to the docker socket hosting the remote docker API.
```

Comment:
nit: insert line break before `Path...` the `util.Markdown` command will strip empty lines and remove consistent indentation... The first line here is not indented like the second line... nit.. btw, if you have better ideas for a macro we could use to write markdown I'm all ears (but maybe we tweak that in different PR)